src/firefox/preferences.js

```diff
@@ -110,7 +110,7 @@ export function getPrefs(
 ): FirefoxPreferences {
   const appPrefs = prefs[app];
   if (!appPrefs) {
-    throw new WebExtError(`Unsupported application: ${app}`);
+    throw new UsageError(`Unsupported application: ${app}`);
```

Comment:
This should be a `WebExtError` because the developer can't select the type of Firefox application yet. When we support Android this might change but this is an internally used function anyway so probably not.